en/firefox/whatsnew/whatsnew-developer-mdnplus.ftl

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/102.0a2/whatsnew/
+
+# HTML page title
+developer-mdnplus-page-title = { -brand-name-firefox-developer-edition }
+developer-mdnplus-congrats-you-now-have-latest = Congrats. You now have the latest version of { -brand-name-firefox-browser } { -brand-name-developer-edition }.
+
+# Main title
+developer-mdnplus-more-mdn-your-mdn = More { -brand-name-mdn }. <em>Your</em> { -brand-name-mdn }.
+developer-mdnplus-mdn-is-an-open-source = { -brand-name-mdn-web-docs } is an open-source, collaborative project documenting Web platform technologies, including CSS, HTML, JavaScript and Web APIs. We also provide an extensive set of learning resources for beginning developers and students.
+developer-mdnplus-support-mdn-and-make = Support { -brand-name-mdn } <em>and</em> make it your own.
+
+# CTA button
+developer-mdnplus-get-started = Get Started
+
+developer-mdnplus-whats-included = What’s included
```

Comment:
Nothing wrong with the text. On the staging server, there is an underscore at the end of this string. Double check the code.

Reply:
That's a purely visual flourish added by CSS, it's not in the text. I think it's to make it look programmer-y?

Reply:
OK, approved then.